packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -35,7 +35,7 @@ const submitLambda = new lambda.Function(this, 'SubmitLambda', { ... });
 const getStatusLambda = new lambda.Function(this, 'CheckLambda', { ... });
 
 const submitJob = new sfn.Task(this, 'Submit Job', {
-    task: new tasks.InvokeFunction(submitLambda),
+    task: new tasks.RunLambdaTask(submitLambda),
```

Comment:
Given that a docs section below says the following:

```
Except `tasks.InvokeActivity`, the [service integration
pattern](https://docs.aws.amazon.com/step-functions/latest/dg/connect-to-resource.html)
(`integrationPattern`) are supposed to be given as parameter when customers want
to call integrated services within a Task state. The default value is `FIRE_AND_FORGET`.
```

Either the docs are wrong or these two calls cannot be equivalent.

The original one (`InvokeFunction`) would call a Lambda function, wait for it to finish and add the Lambda's return value to the state object.

If the default integration pattern is indeed `FIRE_AND_FORGET`, then this call will now call `Invoke`, add the [response object of `Invoke`](https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html#API_Invoke_ResponseElements) to the state object, and have moved on to the next state by the time the Lambda function finishes.

By the way, I think that `FIRE_AND_FORGET` is a silly default. How many people will be writing StepFunctions state machines to fire off tasks one after the other and not care about the result? Versus how many are doing it to stitch together long-running operations where they DO care about the outcome? `SYNC` would make a much better default for nearly all cases, methinks.

Reply:
good point, i'll update the example.

I agree with the sentiment on the default - any thoughts there @wqzoww